indico/queries/workflow.py

```diff
@@ -304,7 +311,8 @@ def __init__(
             submission: bool = True,
             bundle: bool = False,
             result_version: str = None,
-            streams: Dict[str, io.BufferedIOBase] = None
+            streams: Dict[str, io.BufferedIOBase] = None,
+            raw_text: str = ""
```

Comment:
I would suggest changing "raw text" to "text". And note in the comment it may still go through OCR. once the platform has updated support we can change the comment to match what the platform will do and skip the temp file.